dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControlSimp.mo

```diff
@@ -0,0 +1,83 @@
+within Dynawo.Electrical.Controls.Voltage;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model SecondaryVoltageControlSimp "Model for simplified secondary voltage control"
+  import Modelica;
+  import Dynawo.Types;
+
+  //Regulation parameters
+  parameter Types.PerUnit Ck "Sensitivity of voltage to level variations in pu (base UNom)";
+  parameter Types.PerUnit DnsDtm "Level slope limitation in pu/min (base QNomAlt)";
+  parameter Types.Time tau "PI time constant in s";
```

Comment:
Are DnsDtm and tau the real names in the control? And so it's why we use them or are there names in Eurostag? If it is just Eurostag names we can have better ones perhaps.

Reply:
I need to double check. I can come up with better names.